tests/test_app.py

```diff
@@ -0,0 +1,62 @@
+import unittest,json
+
+from WeConnect.app import app
+
+
+class BasicsTestCase(unittest.TestCase):
+    def setUp(self):
+        self.app = app
+        self.dummy_browser = self.app.test_client()
+        
+        self.user_data = {"firstname": "Whitney", "username": "Budhi", "password": "2323"}
+        self.business_data = {"businessname": "Nakumatt", "description": "blahblahblah", "location": "Nairobi"}
+
+
+    def test_create_user(self):
+        response = self.dummy_browser.post(
+            'api/auth/register',
+            headers={'Content-Type': 'application/json'},
+            data=json.dumps(self.user_data))
+        result = json.loads(response.data.decode())
+        self.assertEqual(result["message"], "User created successfully")
+
+    def test_login_user(self):
+        response = self.dummy_browser.post(
+            'api/auth/login',
+            headers={'Content-Type': 'application/json'},
+            data=json.dumps(self.user_data))
+        result = json.loads(response.data.decode())
+        self.assertEqual(result["message"], "You are successfully logged in")

```

Comment:
Consider also testing for the contents of the response. ie for the access token.